web/src/components/atoms/Status/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "@emotion/styled";
+import { CSSProperties } from "react";
+
+type StatusProps = {
+  status: string;
+  color: CSSProperties["color"];
+};
+
+const Status: React.FC<StatusProps> = ({ status, color }) => {
+  return (
+    <StatusWrapper>
+      <Circle color={color} />
```

Comment:
It's better to use `antd badge` component.
To be more consistent across the system, also portable examples use it.

Reply:
Done!